member-service/src/main/java/com/hedvig/memberservice/services/NorwegianBankIdService.kt

```diff
@@ -36,7 +36,7 @@ class NorwegianBankIdService(
         )
     }
 
-    fun sign(memberId: String, ssn: String) =
+    fun sign(memberId: String, ssn: String?) =
```

Comment:
```suggestion
    fun sign(memberId: String, ssn: String) =
```
is the hypothesis that we will have the ssn?